commit 59711f9ec219bf5245a8e95989803fb503adc52d
Author: Ilya Dryomov <idryomov@gmail.com>
Date:   Mon Oct 26 17:01:53 2020 +0100

    libceph: amend cephx init_protocol() and build_request()
    
    In msgr2, initial authentication happens with an exchange of msgr2
    control frames -- MAuth message and struct ceph_mon_request_header
    aren't used.  Make that optional.
    
    Stop reporting cephx protocol as "x".  Use "cephx" instead.
    
    Signed-off-by: Ilya Dryomov <idryomov@gmail.com>

diff --git a/include/linux/ceph/ceph_fs.h b/include/linux/ceph/ceph_fs.h
index d44d98033d58..6d986e52000b 100644
--- a/include/linux/ceph/ceph_fs.h
+++ b/include/linux/ceph/ceph_fs.h
@@ -95,6 +95,7 @@ struct ceph_dir_layout {
 
 #define CEPH_AUTH_UID_DEFAULT ((__u64) -1)
 
+const char *ceph_auth_proto_name(int proto);
 
 /*********************************************
  * message layer